strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/resources/etc/conf/strongbox-cron-tasks.yaml

```diff
@@ -7,7 +7,7 @@ cronTasksConfiguration:
       oneTimeExecution: false
       immediateExecution: false
     - uuid: 9dd9a6ef-d953-4ce0-be3d-3031fcfd993b
-      name: Re-generate Artifacts Checksums Cron Job
+      name: Re-create Artifacts Checksums Cron Job
```

Comment:
Please, change this to:
```
      name: Re-create artifact checksums cron job
```

(This should possibly also be changed for the productio YAML file as well).

Reply:
sorry, it was an accident. It shouldn't be renamed. I'll revert this change.